inst/pages/miaverse.qmd

```diff
@@ -115,28 +115,38 @@ analysis and visualization.
 
 ### SE supporting packages {#sec-sub-diff-abund}
 
-The following DA methods support `(Tree)SummarizedExperiment`.
+The following methods support `(Tree)SummarizedExperiment`.
 
-- [ANCOMBC](https://bioconductor.org/packages/devel/bioc/html/ANCOMBC.html) [@ancombc2020] for differential abundance analysis
-- [benchdamic](https://bioconductor.org/packages/release/bioc/vignettes/benchdamic/inst/doc/intro.html) [@Calgaro2022] for benchmarking differential abundance methods
 - [ALDEx2](https://www.bioconductor.org/packages/release/bioc/html/ALDEx2.html) [@Gloor2016] for differential abundance analysis
-
-### Other relevant packages
-
+- [ampliseq](https://nf-co.re/ampliseq) (@straub2020) for amplicon sequencing analysis workflow using DADA2 and QIIME2
+- [ANCOMBC](https://bioconductor.org/packages/devel/bioc/html/ANCOMBC.html) [@ancombc2020] for differential abundance analysis
+- [benchdamic](https://www.bioconductor.org/packages/release/bioc/vignettes/benchdamic/inst/doc/intro.html) [@Calgaro2022] for benchmarking differential abundance methods
+- [curatedMetagenomicData](https://bioconductor.org/packages/release/data/experiment/html/curatedMetagenomicData.html) contains standardized curated human microbiome data for novel analyses.
+- [holofoodr](https://www.bioconductor.org/packages/release/bioc/html/HoloFoodR.html) for interfacing EBI HoloFood resource
+- [IntegratedLearner](http://htmlpreview.github.io/?https://github.com/himelmallick/IntegratedLearner/blob/master/vignettes/IntegratedLearner.html) for multiomics classification and prediction
+- [iSEEtree](https://microbiome.github.io/iSEEtree/articles/iSEEtree.html) (@Benedetti2024iseetree) for interactive visualisation of hierarchical data
+- [lefser](https://www.bioconductor.org/packages/release/bioc/html/lefser.html) [@Asya2024] for metagenomic biomarker discovery
+- [maaslin](https://bioconductor.org/packages/devel/bioc/html/maaslin3.html) (@Nickols2024) for differential abundance analysis
```

Comment:
I think this list looks good!!

Check that the naming is exact.

maaslin --> MaAsLin 3